### ISSUE TITLE
ModelSummaryToSacredInfo: Some of the entries of the run were not BSON-serializable

### ISSUE THREAD

**Issue by bzamecnik** [bzamecnik/melon#1]
At the end of training when observing to MongoDB.

```
Warning: Some of the entries of the run were not BSON-serializable!
 They have been altered such that they can be stored, but you should fix your experiment!Most likely it is either the 'info' or the 'result'.
WARNING - pitch_duration_melody - The observer '<sacred.observers.mongo.MongoObserver object at 0x7f43f49482b0>' failed at some point during the run.
```

**Comment by bzamecnik** [comment]
It turned out that `model.total_params()` in Keras returns value of type `numpy.int64`.

**Comment by bzamecnik** [comment]
Fixed in version 0.1.2.
